<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="0" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-06-09T14:47:47 IST" name="Suite" finished-at="2022-06-09T14:49:45 IST" duration-ms="118584">
    <groups>
      <group name="Regression">
        <method signature="AccountCreationPageTest.validateAccountCreateTest(java.util.HashMap)[pri:2, instance:com.ecommerce.testCases.AccountCreationPageTest@42e99e4a]" name="validateAccountCreateTest" class="com.ecommerce.testCases.AccountCreationPageTest"/>
      </group> <!-- Regression -->
    </groups>
    <test started-at="2022-06-09T14:47:47 IST" name="ChromeTest" finished-at="2022-06-09T14:49:45 IST" duration-ms="118584">
      <class name="com.ecommerce.testCases.AccountCreationPageTest">
        <test-method is-config="true" signature="beforeSuite()[pri:0, instance:com.ecommerce.testCases.AccountCreationPageTest@42e99e4a]" started-at="2022-06-09T14:47:46 IST" name="beforeSuite" finished-at="2022-06-09T14:47:47 IST" duration-ms="240" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.ecommerce.testCases.AccountCreationPageTest@42e99e4a]" started-at="2022-06-09T14:47:47 IST" name="setup" finished-at="2022-06-09T14:47:57 IST" duration-ms="9723" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateAccountCreateTest(java.util.HashMap)[pri:2, instance:com.ecommerce.testCases.AccountCreationPageTest@42e99e4a]" started-at="2022-06-09T14:47:57 IST" name="validateAccountCreateTest" data-provider="newAcountDetailsData" finished-at="2022-06-09T14:48:14 IST" duration-ms="17013" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Company=meslova, Email=fsfsfdsf@gmail.com, Address=hytech, FirstName=TestUser, Gender=Mrs, City=San, MobilePhone=9556238894, Month=7, Year=1985, State=Alabama, Zipcode=91436, Country=United States, LastName=UserTest, SetPassword=test@123, Day=22}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [http://automationpractice.com/index.php?controller=authentication] but found [http://automationpractice.com/index.php?controller=my-account]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [http://automationpractice.com/index.php?controller=authentication] but found [http://automationpractice.com/index.php?controller=my-account]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.ecommerce.testCases.AccountCreationPageTest.validateAccountCreateTest(AccountCreationPageTest.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateAccountCreateTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.ecommerce.testCases.AccountCreationPageTest@42e99e4a]" started-at="2022-06-09T14:48:14 IST" name="tearDown" finished-at="2022-06-09T14:48:15 IST" duration-ms="790" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.ecommerce.testCases.AccountCreationPageTest@42e99e4a]" started-at="2022-06-09T14:48:15 IST" name="setup" finished-at="2022-06-09T14:48:23 IST" duration-ms="8336" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateAccountCreateTest(java.util.HashMap)[pri:2, instance:com.ecommerce.testCases.AccountCreationPageTest@42e99e4a]" started-at="2022-06-09T14:48:23 IST" name="validateAccountCreateTest" data-provider="newAcountDetailsData" finished-at="2022-06-09T14:48:49 IST" duration-ms="25444" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Company=tcs, Email=agwv@gmail.com, Address=madhpur, FirstName=TestUser, Gender=Mrs, City=San, MobilePhone=7077777607, Month=7, Year=1985, State=Alabama, Zipcode=91436, Country=United States, LastName=UserTest, SetPassword=test@124, Day=22}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [http://automationpractice.com/index.php?controller=authentication] but found [http://automationpractice.com/index.php?controller=my-account]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [http://automationpractice.com/index.php?controller=authentication] but found [http://automationpractice.com/index.php?controller=my-account]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.ecommerce.testCases.AccountCreationPageTest.validateAccountCreateTest(AccountCreationPageTest.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateAccountCreateTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.ecommerce.testCases.AccountCreationPageTest@42e99e4a]" started-at="2022-06-09T14:48:49 IST" name="tearDown" finished-at="2022-06-09T14:48:50 IST" duration-ms="798" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.ecommerce.testCases.AccountCreationPageTest@42e99e4a]" started-at="2022-06-09T14:48:50 IST" name="setup" finished-at="2022-06-09T14:49:05 IST" duration-ms="15485" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateAccountCreateTest(java.util.HashMap)[pri:2, instance:com.ecommerce.testCases.AccountCreationPageTest@42e99e4a]" started-at="2022-06-09T14:49:05 IST" name="validateAccountCreateTest" data-provider="newAcountDetailsData" finished-at="2022-06-09T14:49:44 IST" duration-ms="38674" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Company=cts, Email=dfdagwv@gmail.com, Address=ammerpet, FirstName=TestUser, Gender=Mrs, City=San, MobilePhone=9777997046, Month=7, Year=1985, State=Alabama, Zipcode=91436, Country=United States, LastName=UserTest, SetPassword=test@125, Day=22}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [http://automationpractice.com/index.php?controller=authentication] but found [http://automationpractice.com/index.php?controller=my-account]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [http://automationpractice.com/index.php?controller=authentication] but found [http://automationpractice.com/index.php?controller=my-account]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.ecommerce.testCases.AccountCreationPageTest.validateAccountCreateTest(AccountCreationPageTest.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateAccountCreateTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.ecommerce.testCases.AccountCreationPageTest@42e99e4a]" started-at="2022-06-09T14:49:44 IST" name="tearDown" finished-at="2022-06-09T14:49:45 IST" duration-ms="809" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="afterSuite()[pri:0, instance:com.ecommerce.testCases.AccountCreationPageTest@42e99e4a]" started-at="2022-06-09T14:49:45 IST" name="afterSuite" finished-at="2022-06-09T14:49:46 IST" duration-ms="455" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- com.ecommerce.testCases.AccountCreationPageTest -->
    </test> <!-- ChromeTest -->
  </suite> <!-- Suite -->
</testng-results>
